### PR TITLE
Add extension maturity table

### PR DESCRIPTION
**Pull request by tariqkurd-repo**
Clarify which extensions are mature, and which are still subject to investigation

### CODE REVIEW

**Review by arichardson — APPROVED** [review]

File: src/introduction.adoc [hunk 1]
```diff
@@ -90,13 +90,28 @@ RISC-V.
 It also adds a mode bit in the encoding of capabilities to
 allow changing the current CHERI execution mode using indirect jump
 instructions.
+{sh4add_ext_name}::   Addition of <<SH4ADD>> and <<SH4ADD_UW>> as the natural datatype is 16-bytes.
```
Comment (arichardson) [inline src/introduction.adoc:93]:
```suggestion
{sh4add_ext_name}::   Addition of <<SH4ADD>> and <<SH4ADD_UW>> as CHERI capabilities are 16 bytes when XLEN=64.
```
Maybe this is clearer?

File: src/introduction.adoc [hunk 2]
```diff
@@ -90,13 +90,28 @@ RISC-V.
 It also adds a mode bit in the encoding of capabilities to
 allow changing the current CHERI execution mode using indirect jump
 instructions.
```
Comment (andresag01) [inline src/introduction.adoc:92]: We should delete, or at least modify, this sentence which is old. We also support changing the mode with modesw which is more efficient than the indirect jumps

File: src/introduction.adoc (hunk 1 above)
Comment (tariqkurd-repo) [reply src/introduction.adoc:93]:
```suggestion
{sh4add_ext_name}::   Addition of <<SH4ADD>> and <<SH4ADD_UW>> for RV64 only, as CHERI capabilities are 16 bytes when XLEN=64
```

File: src/introduction.adoc (hunk 2 above)
Comment (tariqkurd-repo) [reply src/introduction.adoc]: I've removed that text

**Pull request merged**